packages/assets-controllers/src/token-prices-service/abstract-token-prices-service.ts

```diff
@@ -7,6 +7,24 @@ export type TokenPrice<TokenAddress extends Hex, Currency extends string> = {
   tokenAddress: TokenAddress;
   value: number;
   currency: Currency;
+  allTimeHigh?: number;
```

Comment:
Do all of these properties need to be optional? It seems like this would add unnecessary logic to code that wants to make use of them, as they'd constantly need to check for `undefined`. See my notes in CodefiV2 on how we could avoid this.